src/libaktualizr-posix/asn1/messages/ipuptane_message.asn1

```diff
@@ -31,10 +31,20 @@ IpUptane DEFINITIONS ::= BEGIN
 		...
 	}
 
-	AKDiscoveryReqMes ::= SEQUENCE {
-		port INTEGER(0..65535),
-		...
-	}
+  AKGetInfoReqMes ::= SEQUENCE {
+    ...
+  }
+
+  AKGetInfoRespMes ::= SEQUENCE {
+    ecuSerial UTF8String,
+    hwId UTF8String,
+    ...
+  }
+
+  AKDiscoveryReqMes ::= SEQUENCE {
+    port INTEGER(0..65535),
+    ...
+  }
```

Comment:
Not critical at all, but the indenting here is inconsistent.

Reply:
I am not sure if I see it.

Reply:
If you look at the surrounding lines on github, it is more apparent. I think it's a tab vs. spaces thing.

Reply:
Yeah, a tab is used for indentation here while my editor inserts spaces on tap pressing.